src/js/core/widget/core/PanelChanger.js

```diff
@@ -52,6 +52,7 @@
 				page = ns.widget.core.Page,
 				panel = ns.widget.core.Panel,
 				events = ns.event,
+				COOKIE_MAX_AGE = 24 * 60 * 60, // one day;
```

Comment:
Wouldn't be user surprised that his values gone after one day?. There is no expiration date for localStorage. This changes behavior a bit.

Reply:
This arbitrary value but seems unnecessary in this case. The cookie should expired with session.